{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/users/apps.py

```diff
@@ -4,3 +4,6 @@
 class CoreConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "{{ cookiecutter.project_slug }}.users"
+
+    def ready(self):
+        import {{ cookiecutter.project_slug }}.users.signals
```

Comment:
Probably can do `from . import signals` for a shorter (and easier to copy) bit of code.

Reply:
Changed to a relative import 🚀!!